site/src/pages/tutorials/tutorial-blog-rest-step6.mdx

```diff
@@ -0,0 +1,160 @@
+---
+type: tutorial
+tags:
+  - rest
+  - request
+  - request-converter
+  - exceptions
+  - exception-handling
+  - blocking
+level: basic
+---
+
+# Step 6. Implement DELETE operation
+
+In this step, we will write a method for updating blog posts. Two new things we will try in this section. One is blocking and the other is exception handling. 
+
+// TODO: TW 아래 내용은 다른 형식으로 제공할 수 있음. 고민 중임.
+
+<Tip>
+
+View the sample service code [here](#).
+
+</Tip>
+
+// TODO: TW 이 부분은 별도 섹션이 아닌 adomination으로 변경될 가능성 있음. 고민 중임.
+// TODO: List update 예정
+
+After completing this step, you will have learnt to use the following Armeria features:
+
+  - HTTP method mapping (<type://@Delete>)
+  - Parameter injection (<type://@Param>)
+  - Exception handler (TBU)
+  - Blocking task executor (<type://@Blocking)
```

Comment:
```suggestion
  - Blocking task executor (<type://@Blocking>)
```
Just in case you missed it. 😉

Reply:
Thank you, :) The list has been moved to somewhere else, but added the closing bracket.